ko/sql/datatype.rst

```diff
@@ -1135,7 +1135,7 @@ CHAR(n)
 
 고정길이 문자열은 **CHAR** (*n*)로 표현하며, 여기서 *n* 은 문자의 개수를 나타낸다. *n* 이 생략되면 길이는 기본값인 1로 지정된다. 
 
-문자열의 길이가 *n* 을 초과하면 초과 부분을 절삭한다. *n* 보다 작은 문자열이 저장되면 나머지 부분은 공백 문자로 채워진다.
+문자열의 길이가 *n* 을 초과하면 **allow_truncated_string** 설정 값이 **yes** 인 경우 초과 부분을 절삭하지만, 설정 값이 **no** 인경우 오류가 발생한다. *n* 보다 작은 문자열이 저장되면 나머지 부분은 공백 문자로 채워진다.
```

Comment:
To translate error, 에러 is used instead of 오류 on this page. please see 1654
```suggestion
문자열의 길이가 *n* 을 초과하면 **allow_truncated_string** 설정 값이 **yes** 인 경우 초과 부분을 절삭하지만, 설정 값이 **no** 인경우 에러가 발생한다. *n* 보다 작은 문자열이 저장되면 나머지 부분은 공백 문자로 채워진다.
```